src/Translator.php

```diff
@@ -90,7 +90,8 @@ public function __construct(
      */
     protected function getMessage($key)
     {
-        if ($message = ($this->package->getMessage($key))) {
+        $message = $this->package->getMessage($key);
+        if ($message) {
```

Comment:
Nevermind, i just spotted the same test in the `translate` method when you replace a [not found translation by the key](https://github.com/auraphp/Aura.Intl/blob/2.x/src/Translator.php#L130).